lib/http.js

```diff
@@ -27,6 +27,7 @@ const {
   ObjectDefineProperty,
 } = primordials;
 
+const { CloseEvent, WebSocket, MessageEvent } = require('internal/deps/undici/undici');
```

Comment:
Can you require these dynamically on first usage? Undici is quite big.

Reply:
yes. good point.

Reply:
Docs pending and on it.